docs/components/sidebar-nav.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Sidebar with Navigation
+summary: Contains and organizes all of the navigation for your web app.
+tags: components
+layout: guide
+eleventyNavigation:
+  key: Sidebar with Navigation
+  parent: Components
+  order: 270
+  excerpt: Contains and organizes all of the navigation for your web app.
+  img: /img/illustrations/illus-share-social.svg
+---
+  
+## Best Practice
+
+The first child element of the `id=page-wrapper` is the Sidebar with Navigation. This is the `nav`. For small devices like smartphones, it makes responsive adjustments. Add Top Level and Child Level items as needed.
+
+- Used in the [App Screen](/components/app-screen/).
+- Presents only two levels of navigation.
+- Current pages should get the class `active` added to the class list of the `li` and `a` which hold the link.
```

Comment:
This only applies to the child pages at the present moment (no indicator changes on `sidebar-item`); might want to also mention that the `sidebar-submenu` does not get revealed unless its display goes to `block`.

Reply:
noted in a bullet item to come